index.js

```diff
@@ -911,62 +910,6 @@ const ContainerService = {
     return Promise.resolve(identity ? this._convert(identity) : null);
   },
 
-  createIdentity(args) {
-    this.sendTelemetryPayload({
-      "event": "add-container",
-    });
-
-    for (let arg of [ "name", "color", "icon"]) { // eslint-disable-line prefer-const
-      if (!(arg in args)) {
-        return Promise.reject("createIdentity must be called with " + arg + " argument.");
-      }
-    }
-
-    const color = this._fromNameToColor(args.color);
-    const icon = this._fromNameToIcon(args.icon);
```

Comment:
Convert still uses these. I think what I have on my laptop removes this through. Will submit Monday